docs/docs/multipart.md

```diff
@@ -0,0 +1,261 @@
+# Multipart and Form Handling
+
+Multipart is a content type for messages (requests or responses) composed of multiple parts.
+Each part is itself similar to a message in that it has its own body and headers. A common use
+for multipart requests is for user-submitted forms, especially ones that include files.
+Browsers also support (and default to) another encoding for forms, `application/x-www-form-urlencoded`.
+This encoding is simpler, but is not often used for binary data.
+
+## UrlForm
+
+To handle `application/x-www-form-urlencoded` messages, http4s provides `UrlForm` and respective `EntityEncoder` and 
+`EntityDecoder`. The following example shows the client sending a form request and a server parsing it:
+
+```scala mdoc:silent
+import org.http4s.client.Client
+import cats.effect._
+import org.http4s._
```

Comment:
Just a nitpick, but we have different notations for the wildcard import over this page 🥸

Reply:
Do you mean the scala-cli block? I rewrote them just now